R/reportFE.R

```diff
@@ -539,7 +539,15 @@ reportFE <- function(gdx, regionSubsetList = NULL,
 
   p36_floorspace <- readGDX(gdx, "p36_floorspace", react = "silent")[, t, ]
   if (!is.null(p36_floorspace)) {
-    out <- mbind(out, setNames(p36_floorspace, "Energy Service|Buildings|Floor Space (bn m2/yr)"))
```

Comment:
Do I understand that correctly that `p36_floorspace ` is either one single component (`buildings`) or three, and the code is flexible enough to handle both situations?

Reply:
Yes, the parameter is just passed through to get here. See [#1197](https://github.com/remindmodel/remind/pull/1197)